todo_app.js

```diff
@@ -63,15 +75,28 @@
          }*/
     }
 
-    function renderTodo(todoObject) {
+    function renderTodos(list) {
         // return string: <li><input type="checkbox" checked or
         // unckecked if todoObject.checked is true> todo text</li>
+        var lsTodos = [];
+        lsTodos = TodoManager.loadTodos();
+        lsTodos.map(function(todo) {
+            return renderTodo(todo);
+        })
     }
 
-    function renderTodos(todosList) {
+    function renderTodo(todo) {
         // uses renderTodo for each todo in list and returns contcatenated string
+        return '<li><label><input type="checkbox">' + todo.text + '</label></li>'
     }
 
+    renderList();
```

Comment:
Why are you rendering list before app is initialized? This is why you have `undefined` element is the beginning of the list. All initial logic should be placed in `initApp()`